.gitignore

```diff
@@ -36,6 +36,9 @@ src/modbus-version.h
 src/win32/modbus.dll.manifest
 tests/unit-test.h
 
+# Generated by meson build
+/build
```

Comment:
https://mesonbuild.com/FAQ.html#how-do-i-ignore-the-build-directory-in-my-vcs

You don't need to add this, Meson does it magically for you. :)

Reply:
That's interesting, however, I just tested the claim made by the meson FAQ, expecting it to create a `build/.gitignore` file and none was created.

Reply:
Well... of course this is because I have an older version of meson installed :smile: 
```
$ meson --version
0.53.2
```